tests/common.ts

```diff
@@ -24,6 +25,35 @@ export const createAccount = async (page: Page, seed: string, name: string): Pro
   await page.getByRole('button', { name: 'Add the account with the supplied seed' }).click();
 };
 
+export const connectToNetwork = async (page: Page): Promise<void> => {
```

Comment:
Correct me if I am wrong, but I recall we have methods below in common, why moving them to test file?

Reply:
Sorry, I didn't get your point fully. I just changed the location to separate the metamask related code.

Reply:
I am sorry. My mistake, somehow I figured out that the code is in àssets.spec.ts`